src/main/resources/db/migration/V1__create_users_table.sql

```diff
@@ -1,5 +1,25 @@
 CREATE TABLE users (
-  id INTEGER PRIMARY KEY,
```

Comment:
Migration files shouldn't be updated after its creation, you have to create a brand new file with the changes

Reply:
I try it many times but couldn't run because mis match error.
I will try to do it again